src/app/modules/ix-forms/services/form-error-handler.service.ts

```diff
@@ -66,13 +66,16 @@ export class FormErrorHandlerService {
           .find((controlOfArray) => isExactMatch(errorMessage, controlOfArray.value));
       }
 
-      control.setErrors({
-        manualValidateError: true,
-        manualValidateErrorMsg: errorMessage,
-        ixManualValidateError: { message: errorMessage },
-      });
-
-      control.markAsTouched();
+      if (!control) {
```

Comment:
Similar to line 55?

Reply:
This follows another check for specific array item where we compare value instead of looking for the field by name in the form group.